tasks/main.yml

```diff
@@ -1,6 +1,8 @@
 ---
 # Variable configuration.
 - include: variables.yml
+- include: cloudhost.yml
```

Comment:
make this 'mysql.yml'.  we're already in the cloudhost role so to have a 'cloudhost' include in the cloudhost role is weird.